recipes/default.rb

```diff
@@ -17,6 +17,8 @@
 # limitations under the License.
 #
 
+::Chef::Recipe.send(:include, RsyslogCookbook::Helpers)
```

Comment:
Instead of including into `Chef::Recipe` and making it available everywhere you can use extend to have it only included in this recipe

```
extend RsyslogCookbook::Helpers
```